README.md

```diff
@@ -1,7 +1,17 @@
 # Wikijump
 
-----
+Wikijump is the [SCP Wiki](http://www.scpwiki.com/)'s fork of the unmaintained [Wikidot](https://github.com/gabrys/wikidot).
+It aims to significantly refactor legacy code, remove misfeatures and dead code, and implement new useful features and bugfixes that are needed in mainline Wikidot.
 
-## About
+This project is being developed by the SCP Wiki's [Technical Team](http://05command.wikidot.com/technical-staff-main) as part of Project Foundation.
+Issues are tracked on our [JIRA](https://scuttle.atlassian.net/browse/WDBUGS).
 
-Wikijump is a platform for creating wikis, based on [Wikidot](http://wikidot.com/).
+Questions, comments or suggestions can be suggested in the [Feature Requests forum](http://www.scpwiki.com/forum/c-3335630/feature-requests), or in [`#site11` on SkipIRC](http://www.scpwiki.com/chat-guide).
+
```

Comment:
```suggestion

## Contributing

We welcome any suggestions or requests, which can be made in our [Feature Requests forum](http://www.scpwiki.com/forum/c-3335630/feature-requests).

If you would like to offer to volunteer some of your time to Wikijump development in the future, please post in one of our [Recruitment threads](http://scp-wiki.wikidot.com/forum/c-3335629/recruitment). We may request your help when we need it.

```

Reply:
We should be consistent with these forum links. Either `www.scpwiki.com` or `scp-wiki.wikidot.com`

Reply:
Oops! My bad - fixed the suggestion